API/Controllers/ProjectController.cs

```diff
@@ -330,22 +330,53 @@ public async Task<IActionResult> CreateProjectAsync([FromBody] ProjectResource p
                 return BadRequest(problem);
             }
 
-            if(projectResource.CallToAction != null)
+            if(projectResource.CallToActions != null)
             {
-                IEnumerable<CallToActionOption> callToActionOptions =
-                    await callToActionOptionService.GetCallToActionOptionFromValueAsync(
-                        projectResource.CallToAction.OptionValue);
-                if(!callToActionOptions.Any())
+                if(projectResource.CallToActions.GroupBy(cta => cta.OptionValue)
+                                  .Any(cta => cta.Count() > 1))
                 {
                     ProblemDetails problem = new ProblemDetails
-                    {
-                        Title = "Call to action value was not found.",
-                        Detail =
-                                                     "The specified call to action value was not found while creating the project.",
-                        Instance = "40EE82EB-930F-40C8-AE94-0041F7573FE9"
-                    };
+                                             {
+                                                 Title = "Duplicate call to action value.",
+                                                 Detail =
+                                                     "It is not possible to create a project with multiple of the same call to actions.",
+                                                 Instance = "D2C8416A-9C55-408B-9468-F0E5C635F9B7"
+                                             };
                     return BadRequest(problem);
                 }
+
+                //This should be refactored. As for much of the code in the controllers, see issue #411 - Refactor controllers.
+                int maxAmountOfCallToActions = 4;
```

Comment:
It's no big issue, but I think it would be slightly better to have one central place for this variable. Someone could change this value right now but forget to change the same variable in the other method.

Reply:
Completely agree, however I did not specifically know where to place this for now, and as I am researching the refactoring of the controllers as well, I wanted to integrate that problem with it. If you have any suggestions where to place that line for now, I'll be happy to implement them!